style.css

```diff
@@ -1,340 +1,1338 @@
+    color: rgba(231, 238, 247, 0.66);
+}
+
+.breadcrumbs__item::after {
+    content: '';
+    position: absolute;
+    top: 12px;
+    right: -2px;
+    width: 2px;
+    height: 16px;
+    background-color: #D0D7FB2B;
+}
+
+.breadcrumbs__item:last-child:after {
+    display: none;
+}
+
+.sidebar {
+    flex-basis: 25%;
+    margin-right: 32px;
+}
+
+.sidebar__block {
+    padding: 24px;
+    background-color: rgba(29, 28, 40, 1);
+}
+
+.blog-content {
+    flex-basis: 75%;
+}
+
+.blog-content__photo-wrapper {
+    display: flex;
+    gap: 24px;
+}
+
+.blog-content p {
+    margin-bottom: 32px;
+}
+
+.blog__list,
+.blog__blockquote,
+.intro__subtitle,
+.blog-content__photo {
+    margin-bottom: 32px;
+}
+
+.blog__list--mark {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    counter-reset: tidbit-counter;
+    list-style-type: none;
+}
+
+.blog__list li {
+    padding-left: 30px;
+    position: relative;
+}
+
+.blog__list--mark  li {
+    margin-bottom: 8px;
+}
+
+.blog__list--disc li:before {
+    position: absolute;
+    content: '';
+    left: 10px;
+    top: 12px;
+    width: 5px;
+    height: 5px;
+    border-radius: 50%;
+    background-color: var(--color-text);
+}
+
+.blog__list--mark li:before {
+    position: absolute;
+    content: url('images/list-icon.svg');
+    left: 0;
+}
+
+.blog__list--mark li:after {
+    position: absolute;
+    content: counter(tidbit-counter);
+    counter-increment: tidbit-counter;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+}
+
+.blog__text--link {
+    color: #78BBFA;
+}
+
+.blog__blockquote {
+    padding: 16px;
+    background-color: #002754;
+    border-radius: 16px;
+}
+
+.blog__blockquote--text {
+    font-size: 24px;
+    font-weight: 600;
+    line-height: 30px;
+}
+
+p.blog__blockquote--text,
+p.blog__blockquote--rating {
+    margin-bottom: 0;
+}
+
+.blog__blockquote--rating {
+    position: relative;
+    padding-left: 30px;
+}
+
+.blog__blockquote--rating:before {
+    position: absolute;
+    content: url('images/stars.svg');
+    left: 0;
+}
+
+.blog__blockquote--span {
+    padding-right: 10px;
+    opacity: 0.8;
+}
+
+.tags__item a {
+    border: 2px solid rgba(208, 215, 251, 0.17);
+}
+
+.sidebar__title {
+    display: block;
+    margin-bottom: 16px;
+    padding-bottom: 16px;
+    font-family: var(--font-secondary);
+    font-size: 24px;
+    line-height: 32px;
+    border-bottom: 1px solid rgba(191, 197, 238, 0.12);
+}
+
+.categories__item {
+    padding: 6px 12px;
+}
+
+.tags__wrapper {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.tags__item {
+    margin-bottom: 8px;
+    margin-right: 8px;
+}
+
+.tags__item a {
+    display: block;
+    padding: 6px 12px;
+}
+
+.blog-footer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 150px;
+}
+
+.blog-footer__wrapper {
+    display: flex;
+    align-items: baseline;
+    column-gap: 20px;
+}
+
+.blog-content .subtitle {
```

Comment:
Please use modificator (.subtitle-center) instead overriding styles from the parent selector

Reply:
done